src/agent/src/mount.rs

```diff
@@ -1380,4 +1460,56 @@ mod tests {
 
         assert!(testfile.is_file());
     }
+
+    #[test]
+    fn test_get_pagesize_and_size_from_option() {
+        let expected_pagesize_str = "2048";
+        let expected_size_str = "107374182";
+        let expected = (expected_pagesize_str, expected_size_str);
+
+        let data = vec![
+            // (input, expected, is_ok)
+            ("size-1=107374182,pagesize-1=2048", expected, false),
```

Comment:
How about a few more tests:

- `size` but no `pagesize`.
- `pagesize` but no `size`.
- `pagesize` specified with invalid data.
- `size` specified with invalid data.
- Missing values (`pagesize=` and `size=`).